MANIFEST.in

```diff
@@ -37,7 +37,7 @@ include CONTRIBUTING.md
 include *.py
 recursive-include contributions *
 recursive-include bin *
-recursive-include doc *.py *.rst Makefile
```

Comment:
The copyright date still needs to be updated

Reply:
Argh!